Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Block_IP.yml

```diff
@@ -0,0 +1,733 @@
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "3"
+      "yes":
+      - "14"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.IP
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 70
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: 0ca5a10c-a607-43ce-8f5d-1308b8db960c
+    type: condition
+    task:
+      id: 0ca5a10c-a607-43ce-8f5d-1308b8db960c
+      version: -1
+      name: Is there an address group provided?
+      description: Is there an address group provided?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "5"
+      "yes":
+      - "4"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: inputs.StaticAddressGroupName
+            iscontext: true
+          right:
+            value: {}
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 170,
+          "y": 410
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 99f457e4-1a30-4314-8bf7-7bb4f9b6a8e6
+    type: title
+    task:
+      id: 99f457e4-1a30-4314-8bf7-7bb4f9b6a8e6
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1680
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 4b2a7ae5-8d45-4e88-86b1-fa514d8d1d07
+    type: regular
+    task:
+      id: 4b2a7ae5-8d45-4e88-86b1-fa514d8d1d07
+      version: -1
+      name: Get the Address Group List
+      description: Lists all address groups.
+      script: '|||prisma-sase-address-group-list'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "9"
+    scriptarguments:
+      folder:
+        complex:
+          root: inputs.Folder
+      group_id:
+        complex:
+          root: inputs.StaticAddressGroupName
+      name:
+        complex:
+          root: inputs.StaticAddressGroupName
```

Comment:
Name and group ID gets the same value. where is the IP? 
![image](https://github.com/demisto/content/assets/61933087/c035e287-0154-45cc-a0e9-aedb4d7021c2)

Reply:
I see its also the same in the Prisma SASE - Block URL playbook

Reply:
I should remove that and forgot - it was a placeholder for the `name` argument that was missing. Good catch